PocketKit/Sources/PocketKit/MyList/SavedItemsList/SavedItemsListViewModel.swift

```diff
@@ -150,18 +150,17 @@ class SavedItemsListViewModel: NSObject, ItemsListViewModel {
         ]
     }
 
-    func trailingSwipeActions(for objectID: NSManagedObjectID) -> [UIContextualAction] {
+    func trailingSwipeActions(for objectID: NSManagedObjectID) -> [ItemContextualAction]? {
```

Comment:
I dig that we're moving away from `UIContextualAction` here. You can adjust return type to non-optional array if you are so inclined ;)